spec/models/queue_item_spec.rb

```diff
@@ -0,0 +1,43 @@
+require 'spec_helper'
+
+describe QueueItem do
+  let(:queue_item) { Fabricate(:queue_item) }
+  let(:video) { queue_item.video }
+  let(:review) { queue_item.review }
+  let(:user) { queue_item.user }
+  let(:category) { video.categories.first }
+
+  it { should belong_to(:video) }
+  it { should belong_to(:user) }
+  it { should belong_to(:review) }
+
+  describe "#video_title" do
+    it "should get the video title" do
```

Comment:
`"should"` in example message does not help the message and in fact should not be used - Read more [here](http://www.betterspecs.org/#should)